packages/@o3r-training/tools/cli/extract-folder-structure/extract-folder-structure.ts

```diff
@@ -0,0 +1,45 @@
+import {lstat, readdir, readFile, writeFile} from 'node:fs/promises';
+import {type FileSystem, getFilesTree} from '@o3r-training/tools';
+import {join, resolve} from 'node:path';
+import {program} from 'commander';
+
+program
+  .description('Extract folder structure')
+  .requiredOption('--files <files>', 'List of files and folder to extract in addition to the path')
+  .option('-r, --root <root>', 'Root of the extraction')
+  .option('-o, --output <output>', 'Output folder path')
+  .parse(process.argv);
+
+const options: any = program.opts();
+const cwd = options.root ? resolve(process.cwd(), options.root) : process.cwd();
+const stringifyBigObject = (jsonObject: any): string => {
```

Comment:
Why do we need that ?

Reply:
In a previous iteration, I reached the maximum character limit and needed a method to handle this use case.

Reply:
I think the webcontainer would not work well with an object so huge it can't be stringified, we should probably go back to the original JSON.stringify implementation